src/operator/tensor/broadcast_reduce_op_value.cc

```diff
@@ -141,19 +141,22 @@ For example::
 .set_attr<FCompute>("FCompute<cpu>", BroadcastCompute<cpu>);
 
 MXNET_OPERATOR_REGISTER_BROADCAST(broadcast_to)
-.describe(R"code(Broadcast an array to a new shape.
+.describe(R"code(Broadcasts the input array to a new shape.
+
+Broadcasting is a mechanism that allows NDArrays to perform different arithmetic operations
```

Comment:
"Broadcasting is a mechanism that allows NDArrays to perform *different* arithmetic operations
 +with arrays of different shapes efficiently without creating multiple copies of arrays"

Can remove the first *different* (marked by *___* in quoted code)